common/js/scroll_into_view.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Scrolls an element into view.
+ * @param {Node} node - The element to scroll into.
+ * @param {ScrollIntoViewOptions} options.
+ * @returns {void}
+ */
+function scrollIntoView(node, options) {
```

Comment:
WDYT about exposing this to user scripts? Currently we/Playwright only expose the CDP `DOM.scrollIntoViewIfNeeded` method, but wrapping this would avoid the hassle of using `evaluate()`.

Not in this PR, of course.

Reply:
It makes sense! People might need this functionality.